pyomo/contrib/doe/examples/fim_doe_tutorial.ipynb

```diff
@@ -1080,69 +1645,85 @@
       "text/plain": [
-       "<Figure size 432x288 with 2 Axes>"
+       "<Figure size 640x480 with 2 Axes>"
       ]
      },
-     "metadata": {
-      "needs_background": "light"
-     },
+     "metadata": {},
      "output_type": "display_data"
     }
    ],
    "source": [
-    "# this problem has three degrees of freedom, to draw heatmap it needs to fix one dimension\n",
+    "# This problem has three degrees of freedom, to draw heatmap it needs to fix one dimension\n",
     "fixed = {\"('T[0.125]','T[0.25]','T[0.375]','T[0.5]','T[0.625]','T[0.75]','T[0.875]','T[1]')\": 300}\n",
     "\n",
     "all_fim.figure_drawing(fixed, ['CA0[0]','T[0]'], 'Reactor case','$C_{A0}$ [M]', 'T [K]' )"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As shown in the Reactor Case - A optimality figure, A-optimality shows that the most informative region is around $C_{A0}=5.0$ M, $T=300.0$ K, while the least informative region is around $C_{A0}=1.0$ M, $T=700.0$ K."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Key Takeaways\n",
+    "\n",
+    "* MBDoE is helpful for guiding decision-making by maximizing information yield in experimental design.\n",
+    "\n",
+    "* FIM allows us to gain information about the data from a mathematical model in an experiment.\n",
```

Comment:
FIM quantifies the information contained in a set of experiments (data) with respect to a mathematical model